### PR TITLE
Update package parsing for secmon on 3.2

### PR DESCRIPTION
**Pull request by bauerj**
This enables the library to parse the 3.2 packages.

There also seem to be a few new messages but I did not add them yet.

The most important bit is that the robot is not incorrectly detected as a pre-3.x client so the parser doesn't expect the wrong robot mode (0) any more.

### CODE REVIEW

**Comment by bauerj** [comment]
Fixes #5.

File: urx/ursecmon.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ class ParserUtils(object):
     def __init__(self):
         self.logger = logging.getLogger("ursecmon")
         self.is_v30 = False
+        self.is_v32 = False
```
Comment (oroulet) [inline urx/ursecmon.py]: this is a bit of a hack, we should have one variable that is set to the version, not a variable per version. Do you feel able to fix this?

Comment (oroulet) [inline urx/ursecmon.py]: this is a bit of a hack, we should have one variable that is set to the version, not a variable per version. Do you feel able to fix this?

Comment (bauerj) [inline urx/ursecmon.py]: Sure, you're right. I'll change that.

**Comment by oroulet** [comment]
Thanks for the patch!  Could you only use one variable, for example selv.version. instead of one variable per version?

**Comment by bauerj** [comment]
@oroulet Done! :smiley:

**Comment by bauerj** [comment]
Now, everyhing pre 3.0 is detected as 0.0, I hope that is okay.

**Comment by bauerj** [comment]
To be more future-proof, this should use something like the `VersionMessage` to detect the version instead of relying on the correct package size. But I guess that is too much for this changeset.

**Comment by oroulet** [comment]
Sounds like a good idea. Not sure it was available when I wrote this ... 6
years ago

**Comment by bauerj** [comment]
Probably not :wink: This features seems to be more recent, maybe it was added in 3.2?

Anyway, thanks for this great library! It will surely save me a lot of work :+1:

**Pull request merged**